model_hub/model_hub/huggingface/_utils.py

```diff
@@ -0,0 +1,26 @@
+import inspect
+from typing import Any, List, Tuple, Union
+
+import torch.nn as nn
+from datasets import Dataset
+
+
+def get_label_list(labels: Union[List[Any], Tuple[Any]]) -> List[Any]:
+    # Get a list of unique labels sorted alphabetically.
+    unique_labels = set()
+    for label in labels:
+        unique_labels.add(label)
+    label_list = list(unique_labels)
+    label_list.sort()
+    return label_list
```

Comment:
changed the method to work for cases where label itself could have multiple elements 
`unique_labels = unique_labels | set(label)`

agree that otherwise `sorted(set(labels))` is much cleaner